worker/worker.go

```diff
@@ -26,6 +26,14 @@ import (
 	"golang.org/x/crypto/blake2b"
 )
 
+const (
+	lockingPriorityRenew   = 100 // highest
+	lockingPriorityFunding = 90
+
+	lockingDurationRenew   = time.Minute
```

Comment:
why the difference in lock durations?

Reply:
These values are quite random but I wanted to give renews an almost excessive amount of time since they do require more round-trips than funding and since it happens a lot less frequently so hitting that limit is not as problematic.

Reply:
works for me